assets/prometheus-k8s/rules.yaml

```diff
@@ -501,6 +501,14 @@ spec:
       for: 15m
       labels:
         severity: critical
+    - alert: OpenShiftStateMetricsDown
```

Comment:
suggestion: as a follow-up, we could add a test verifying this alert doesn't trigger as a this to e2e tests in origin.

cc @paulfantom

Reply:
In origin e2e we were testing (test is temporarily disabled with https://github.com/openshift/origin/commit/f8db150daac4d2693cda473779ca97bfa9176b1b) how many alerts are firing and allowing only watchdog alert to be active. This means that we are also indirectly testing if `OpenShiftStateMetricsDown` is firing.

Reply:
great! :+1: thanks for the verification!